DocumentsFromSnapshotMigration/docker/Dockerfile

```diff
@@ -1,8 +1,17 @@
 # Using same base image as other Java containers in this repo
 FROM amazoncorretto:11-al2023-headless
+
+# Install the AWS CLI in the container
+RUN dnf update -y && dnf install -y aws-cli
```

Comment:
nit: we should add `&& dnf clean all &&  rm -rf /var/cache/dnf` to reduce the image size